app/src/docs/v1.api-spec.yaml

```diff
@@ -949,7 +916,7 @@ paths:
               $ref: '#/components/schemas/Request-Permit'
       responses:
         '200':
-          description: Permit updated successfully
+          description: Permit upserted successfully
```

Comment:
Will be something to converse about, but generally API documentation focuses on the outcome of the action, not how it was implemented. In this case, it may make sense to retain the updated instead of changing to upserted, because it is fundamentally an update, only that you achieve it with an upsert operation instead.